src/ProgressOnderwijsUtils.Analyzers/MustUseExpressionResultAnalyzer.cs

```diff
@@ -69,6 +79,21 @@ static void AnalyzeArrowExpressionClause(SyntaxNodeAnalysisContext context, Arro
             }
         }
 
+        static void AnalyzeLambdaExpression(SyntaxNodeAnalysisContext context, LambdaExpressionSyntax lambda)
+        {
+            var convertedType = context.SemanticModel.GetTypeInfo(lambda).ConvertedType;

```

Comment:
Denk dat het we mee valt, omdat `convertedType` over het algemeen `null` is. Maar ik zal het zeker in de gaten gaan houden ja ...

Reply:
Maar bij al onze fluent-api dingen toch niet?

Sowieso is de SemanticModel best duur, maar he, al doende merken we wel hoe duur het is.